docs/root/configuration/http_conn_man/headers.rst

```diff
@@ -65,12 +65,11 @@ the  :option:`--service-node` option.
 x-envoy-external-address
 ------------------------
 
-It is a common case where a service wants to perform analytics based on the client IP address. Per
-the lengthy discussion on :ref:`XFF <config_http_conn_man_headers_x-forwarded-for>`, this can get
-quite complicated. A proper implementation involves forwarding XFF, and then choosing the first non
-RFC1918 address *from the right*. Since this is such a common occurrence, Envoy simplifies this by
-setting *x-envoy-external-address* during decoding if and only if the request ingresses externally
-(i.e., it's from an external client). *x-envoy-external-address* is not set or overwritten for
+It is a common case where a service wants to perform analytics based on the origin client's IP
+address. Per the lengthy discussion on :ref:`XFF <config_http_conn_man_headers_x-forwarded-for>`,
+this can get quite complicated, so Envoy simplifies this by setting *x-envoy-external-address*
+to the *trusted client address* (as defined in that XFF discussion) if the request is from
```

Comment:
Can you add an anchor there and just link with :ref: rather than with description?